scripts/install.sh

```diff
@@ -206,6 +211,21 @@ while [ "$1" != "" ]; do
         -r | --region )                 shift
                                         region=$1
                                         ;;
+        -llt | --lambda_latency_threshold )    shift
+                                        lambda_latency_threshold=$1
+                                        ;;
+        -a5et | --apigateway5xxerror_threshold )    shift
```

Comment:
You might want to update the usage() function in this script as well with these new options.